### ISSUE TITLE
expo init project proxy error

### ISSUE THREAD

**Issue by ruichen2022** [expo/expo-cli#4262]
### Summary

"expo init newProject" command in Windows 10 command window with admin right behind a proxy server failed to execute.

### Environment

  expo-env-info 1.0.2 environment info:
    System:
      OS: Windows 10 10.0.19043
    Binaries:
      Node: 16.14.2 - C:\Program Files\nodejs\node.EXE
      npm: 8.5.0 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version     2021.1.0.0 AI-211.7628.21.2111.8193401
    Expo Workflow: managed

### Please specify your device/emulator/simulator platform, model and version

Windows 10

### Error output

Err! code ECONNRESET
C:\test>expo init newPrj
√ Choose a template: » blank a minimal app as clean as an empty canvas
× Something went wrong while downloading and extracting the template.
request to https://registry.npmjs.org/expo-template-blank/-/expo-template-blank-44.0.4.tgz failed, reason: read ECONNRESET
FetchError: request to https://registry.npmjs.org/expo-template-blank/-/expo-template-blank-44.0.4.tgz failed, reason: read ECONNRESET
at ClientRequest. (C:\Users\Student\AppData\Roaming\npm\node_modules\expo-cli\node_modules\node-fetch\lib\index.js:1491:11)
at ClientRequest.emit (node:events:526:28)
at TLSSocket.socketErrorListener (node:_http_client:442:9)
at TLSSocket.emit (node:events:526:28)
at emitErrorNT (node:internal/streams/destroy:157:8)
at emitErrorCloseNT (node:internal/streams/destroy:122:3)
at processTicksAndRejections (node:internal/process/task_queues:83:21)

### Reproducible demo or steps to reproduce from a blank project

"expo init newProject" command in Windows 10 command window with admin right behind a proxy server failed to execute.

**Comment by kbrandwijk** [comment]
Please fill out the fields in the issue template (under environment). The information from `npx expo-env-info` will help us troubleshoot your issue.

**Comment by pwh706** [comment]
I have same problem when using windows 10 command and the machine is under company proxy.


D:\react>expo init efg
× Something went wrong while downloading and extracting the template.
request to https://registry.npmjs.org/expo-template-blank/-/expo-template-blank-44.0.4.tgz failed, reason: getaddrinfo ENOTFOUND registry.npmjs.org
FetchError: request to https://registry.npmjs.org/expo-template-blank/-/expo-template-blank-44.0.4.tgz failed, reason: getaddrinfo ENOTFOUND registry.npmjs.org
    at ClientRequest.<anonymous> (C:\Users\pwh706\AppData\Roaming\npm\node_modules\expo-cli\node_modules\node-fetch\lib\index.js:1491:11)
    at ClientRequest.emit (node:events:526:28)
    at TLSSocket.socketErrorListener (node:_http_client:442:9)
    at TLSSocket.emit (node:events:526:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

  expo-env-info 1.0.2 environment info:
    System:
      OS: Windows 10 10.0.19042
    Binaries:
      Node: 16.14.2 - D:\nodejs\node.EXE
      Yarn: 1.22.5 - D:\Yarn\bin\yarn.CMD
      npm: 8.5.0 - D:\nodejs\npm.CMD
    Expo Workflow: managed

**Comment by ruichen2022** [comment]
> Please fill out the fields in the issue template (under environment). The information from `npx expo-env-info` will help us troubleshoot your issue.

  expo-env-info 1.0.2 environment info:
    System:
      OS: Windows 10 10.0.19043
    Binaries:
      Node: 16.14.2 - C:\Program Files\nodejs\node.EXE
      npm: 8.5.0 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version     2021.1.0.0 AI-211.7628.21.2111.8193401
    Expo Workflow: managed


Error Output:
C:\test>expo init newPrj
√ Choose a template: » blank               a minimal app as clean as an empty canvas
× Something went wrong while downloading and extracting the template.
request to https://registry.npmjs.org/expo-template-blank/-/expo-template-blank-44.0.4.tgz failed, reason: read ECONNRESET
FetchError: request to https://registry.npmjs.org/expo-template-blank/-/expo-template-blank-44.0.4.tgz failed, reason: read ECONNRESET
    at ClientRequest.<anonymous> (C:\Users\Student\AppData\Roaming\npm\node_modules\expo-cli\node_modules\node-fetch\lib\index.js:1491:11)
    at ClientRequest.emit (node:events:526:28)
    at TLSSocket.socketErrorListener (node:_http_client:442:9)
    at TLSSocket.emit (node:events:526:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

**Comment by ruichen2022** [comment]
> Please fill out the fields in the issue template (under environment). The information from `npx expo-env-info` will help us troubleshoot your issue.

Hi, any suggestion? Thanks.

**Comment by nick-attlee** [comment]
Ran into the same issue while using version **5.4.3** behind a corporate proxy.  At first I thought the proxy wasn't configured correctly (I set the `http_proxy` and `https_proxy` environment variables in a Windows-based environment), but then I've used the Expo CLI just fine in the past.  And installing Node packages with npm was still working.

Digging a bit deeper I discovered the breaking change for me was from package version **5.0.5** onwards.  Initialising a new Expo project using version **5.0.4** (or earlier) of the CLI worked.

I assume there was a change in **5.0.5** onwards to the way package templates are downloaded? I haven't had time to compare the code changes between versions to see what might have introduced the issue.
